### PR TITLE
TL: first revision for the parallel SDC theory paper

### PR DESCRIPTION
**Pull request by tlunet**

### CODE REVIEW

**Comment by pancetta** [comment]
Huh? You pushed `5.5.1` in #474 already..

File: pySDC/core/sweeper.py [hunk 1]
```diff
@@ -263,3 +263,15 @@ def level(self, L):
     @property
     def rank(self):
         return 0
+
+    def updateVariableCoeffs(self, k):
+        """
+        Eventually update QDelta implicit coefficients if variable ...
```
Comment (brownbaerchen) [inline pySDC/core/sweeper.py:269]: Why `Eventually`? That sounds like it will come in the future

Comment (tlunet) [reply pySDC/core/sweeper.py:269]: Nah, it's just that most preconditioners keep the same coefficient for different `k`

**Comment by brownbaerchen** [comment]
> Huh? You pushed `5.5.1` in #474 already..

That looks like a problem with the diff on GH. I wouldn't worry about it.

**Comment by brownbaerchen** [comment]
Now is the perfect time to automate linting. I updated mine to ruff this morning myself in less than 2 mins using our [guide](https://github.com/Parallel-in-Time/pySDC/blob/master/docs/contrib/02_continuous_integration.md).

File: pySDC/core/sweeper.py (hunk 1 above)
Comment (brownbaerchen) [reply pySDC/core/sweeper.py:269]: Ah, I see! This is a false friend, I guess. https://www.oed.com/search/dictionary/?scope=Entries&q=eventually

Comment (pancetta) [reply pySDC/core/sweeper.py:269]: `Potentially`?

Comment (tlunet) [reply pySDC/core/sweeper.py]: Well, this one was vicious ...

**Pull request merged**